terraform/deployments/ecr/main.tf

```diff
@@ -0,0 +1,112 @@
+# This is supposed to be applied in the production environment
+
+terraform {
+  backend "s3" {
+    bucket  = "govuk-terraform-steppingstone-production"
+    key     = "govuk/ecr.tfstate"
+    region  = "eu-west-1"
+    encrypt = true
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.13"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+
+resource "aws_ecr_repository" "content-store" {
```

Comment:
Do we want one repository per app? Can we not put all the apps in the same repository?

Reply:
That's what I wanted to do originally but that would cause problems with tags as a tag can only exist once per repository.
https://stackoverflow.com/questions/46230838/ecr-what-is-the-registry-and-what-is-a-repository

Reply:
Today I learned! Thanks for explaining that, it makes sense to me now.